src/order/filter/basic.lean

```diff
@@ -709,6 +742,24 @@ by simp only [imp_iff_not_or, eventually_or_distrib_right, not_frequently]
 lemma frequently_top {p : α → Prop} : (∃ᶠ x in ⊤, p x) ↔ (∃ x, p x) :=
 by simp [filter.frequently]
 
+lemma inf_neq_bot_iff_frequently_left {f g : filter α} :
+  f ⊓ g ≠ ⊥ ↔ ∀ {U}, U ∈ f → ∃ᶠ x in g, x ∈ U :=
```

Comment:
Is there any specific reason to mix `∈` and `frequently? E.g., why not use
```suggestion
  f ⊓ g ≠ ⊥ ↔ ∀ {p}, (∀ᶠ x in f, p x) → ∃ᶠ x in g, p x :=
```

Reply:
You're right, that one is a strange mix. Your version does require a bit more massaging in my intended use, but that's ok.